text/2394-async_await.md

```diff
@@ -312,112 +312,58 @@ RFC 提議在前面就包含這兩個構造，因為我們似乎不可避免地
 # 現有技術
 [現有技術]: #prior-art
 
-There is a lot of precedence from other languages for async/await syntax as a
-way of handling asynchronous operation - notable examples include C#,
-JavaScript, and Python.
-在其他語言中，有很多關於 async/await 語法的先例，作為處理非同步操作的一種方式，值得注意的例子包括 C#、JavaScript 和 Python。
+在其他語言中，包含 C#、JavaScript 和 Python，有很多關於使用 async/await 語法作為處理非同步操作的一種方式的先例。
 
-There are three paradigms for asynchronous programming which are dominant
-today:
 目前主流的非同步程式設計有以下三種泛式：
 
-- Async and await notation.
-async 和 await 符號。
-- An implicit concurrent runtime, often called "green-threading," such as
-  communicating sequential processes (e.g. Go) or an actor model (e.g. Erlang).
-  隱式並行運行時間（runtime），通常稱為「綠色執行緒」，例如通信順序進程（例如 Go）或參與者模型（例如 Erlang）。
-- Monadic transformations on lazily evaluated code, such as do notation (e.g.
-  Haskell).
-  延遲執行程式的單元（Monadic）轉換，例如 do 符號（例如 Haskell）。
-
-Async/await is the most compelling model for Rust because it interacts
-favorably with ownership and borrowing (unlike systems based on monads) and it
-enables us to have an entirely library-based asynchronicity model (unlike
-green-threading).
+- async 和 await 符號。
+- 隱式並行運行時間（runtime），通常稱為「綠色執行緒」，例如通信順序進程（例如 Go）或參與者模型（例如 Erlang）。
+- 延遲執行程式的單元（Monadic）轉換，例如 do 符號（例如 Haskell）。
```

Comment:
I might just keep it as is 😂 and also, I might keep do notation as is. or, if we'd like to translate it, it sounds more correct to me to call it `do 語法` instead of `do 符號` 🤔 

```suggestion
- 延遲執行程式中的 Monadic 轉換，例如：Haskell 的 do 語法（do notation）。
```

(I wonder two `例如` in the sentence looks a bit tedious so I changed the wording to be `Haskell 的 do 語法`)